scripts/update-yaml.py

```diff
@@ -13,9 +13,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 #
+import yaml
 import argparse
 import errno
-import yaml
```

Comment:
https://github.com/rcbops/rpc-openstack/issues/595 is where most of it happened.